### DISCUSSION TITLE
Page.get_text() issue

### DISCUSSION THREAD

**Discussion by Mano2731995** [pymupdf/PyMuPDF#2180] [Looking for help] [answered]
Hai jorj

Currently we are using below software requirements in our project(local machine and server same software version) 

1.Python version 3.6.5
2.Pymupdf version 1.19.3
3.fitz version 0.0.1 dev2

Problem scenario:
Page.get_text() concept working in pycharm community edition in local machine but when we deploy the code in dev server page.get_text () line getting problem. 

Kindly give us solution for that issue.

**Comment by JorjMcKie** [comment, marked as answer]
You are using `fitz version 0.0.1 dev2`. **_This package cannot live in the same environment as PyMuPDF!_**
Remove it and install it somewhere else.
Or ask its author for an alternative - *package* `fitz` after all is in its early alpha version state and apparently even is no longer maintained.

PyMuPDF itself **_requires Python 3.7_** or later. I think it is not even possible to compile it for a lower version because it uses language features on both, the Python and the C source level, that are not supported in Python 3.6.
As a general policy, PyMuPDF stops supporting Python versions, when they are no longer maintained by the Python Software Foundation PSF.

**Reply by Mano2731995** [reply]
Thank you for information jorj
